sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.TestLogger/src/Processor/TestProcessor.cs

```diff
@@ -280,10 +280,7 @@ private  void UploadBuffer(string uri, string buffer, string fileRelativePath)
 
         private void UploadBlobFile(string uri, string fileRelativePath, string filePath)
```

Comment:
we can remove this private function as its just empty wrapper

Reply:
okay doing same for  private  void UploadBuffer(string uri, string buffer, string fileRelativePath)
 {
      _blobService.UploadBufferAsync(uri, buffer, fileRelativePath);
 } as it is also empty wrapper.